### ISSUE TITLE
Parts in closed cargo bay have drag [Bug 🐞]: 

### ISSUE THREAD

**Issue by Krazy0** [zer0Kerbal/CargoBays#45]
### Brief description of your issue (aka the problem)

Parts fully enclosed in a cargo bay still have drag. Also the PAW open / closed button always says open in either state.

### Operating system

Win 10

### KSP version

1.12.5

### Version of addon

1.1.99.0

### Steps to reproduce the behavior or issue

In Alt-F12 menu, turn on aero data in PAW. Monitor drag for part inside bay. Drove down to the runway to test. Open/ close the cargo bay. Part shows non-zero drag. 
Did same test with stock and Airplane Plus cargo bays and they show zero drag when closed. 

### Environment

```shell
DaMichel Ltd 0.9.99.0
~80 other mods
```

### Additional context

Airplane Plus seems to work so I'd check that code.

Also had trouble installing with Curseforge Overwolf app... it would not install both DaMichel Ltd and CargoBays. I had to install one to Steam KSP install, copy it to modded KSP install, then repeat for the other one. Maybe it deletes everything in the DaMichel parent folder when installing. Trying to install both at the same time installed only DaMichel/ Cargo Bays and it definitely doesn't work without DaMichelLtd.

Krazy1 on KSP forum

### How did you download and install this?

Other (explain below)

**Comment by zer0Kerbal** [comment]
Thank you @Krazy0!

Will look into - about to update all the DaMichel addons (plan is this week). 

> Also had trouble installing with Curseforge Overwolf app... it would not install both DaMichel Ltd and CargoBays

Yes - either you have to install using the app and copy, or download from the site and copy. *sigh* One of the few known issues with the CF/OW app is the inability to install into subdirectories without wiping contents. I am pressuring CF to resolve this and CF support has been very responsive to my requests and bug reports.

Part of this issue will be resolved when I update the DaMichel addons and push the releases to CurseForge.

**Comment by zer0Kerbal** [comment]
@Krazy0 which parts? specific series (1.25, 1.875, 2.5, 3.75, 5)?

**Comment by zer0Kerbal** [comment]
_might_ be the culprit:

lookupRadius

```
	MODULE
	{	name = ModuleCargoBay
		DeployModuleIndex = 0
		closedPosition = 0
		lookupRadius = 2.865 // 1.61
		partTypeName = Cargo bay
		nodeOuterForeID = top
		nodeOuterAftID = bottom
		nodeInnerForeID = top2
		nodeInnerAftID = bottom2
	}
```

and some `DRAG_CUBES` were missing, incorrect, or duplicated (fixed)

**Comment by zer0Kerbal** [comment]
this release should fix 90% of the issues. if you still have issues - logs kindly and which parts would be very much appreciated.

**Comment by Krazy0** [comment]
I'm using 1.1.99.1 prerelease and I tried a new ship with a 2m long 2.5m bay
The drag is not occluded for parts inside

[KSP (27).log](https://github.com/zer0Kerbal/CargoBays/files/12309718/KSP.27.log)
[Player (12).log](https://github.com/zer0Kerbal/CargoBays/files/12309720/Player.12.log)

**Comment by Krazy0** [comment]
I made a test ship with all the bays except 1.875m. All the parts inside have drag when the doors are closed or open. Surface attached (inside) or node attached all have drag. BTW, from Airplanes Plus "Size 2 Cargo Bay CRG-15" also has drag. Stock bays work OK. Some pics also showing the strange lighting
https://imgur.com/a/qxz9RBp

**Comment by Lisias** [comment]
Hi, @Krazy0 ! I made some tests, and I didn't reproduced your results.

First, I made a "rocket" (easier to check the drag, as the thing accelerates faster) using an CRG-15 cargo bay. Inside the cargo bay there's a FL-TX220 Fuel Tank. Note that the drag is 0.0!

![screenshot28](https://github.com/zer0Kerbal/CargoBays/assets/64334/e1db7d01-c099-49e7-a141-38de81604996)


Now I made the same test, using a MK3 Cargo Bat CRG-25, with a FL-R400 RCS Fuel Tank inside. Please note the drag - it's low, but it's there.

![screenshot27](https://github.com/zer0Kerbal/CargoBays/assets/64334/b870c245-33ce-4bc7-a211-8c409f5a72a1)


So, apaprently, AP's cargo bay is doing a better job than Stock's ones! But there's a trick: you need to open and close the cargo bays before launch, otherwise, yeah, you will get drag on the cargo. It's probably an initialisation problem (I'm working on it).

Crafts used on the test: 
[VAB.CargoBayTests.1125.zip](https://github.com/zer0Kerbal/CargoBays/files/12340355/VAB.CargoBayTests.1125.zip)

**Comment by zer0Kerbal** [comment]
@Krazy0 Thank you - I will review and update.
@Lisias Thank you - and thank you.
There is also the plane vs rocket drag issue as explained in the Dev Diaries for KSP2 (also affects KSP)

**Comment by Lisias** [comment]
We [found the problem](https://forum.kerbalspaceprogram.com/topic/140262-14x-18x-airplane-plus-r264-fixed-issuesgithub-is-up-to-date-dec-21-2019/?do=findComment&comment=4319549)! :)

**Comment by Lisias** [comment]
Nope. WE found the trigger. The problem is some Stock parts being shipped with the wrong definitions.

[CargoB](https://github.com/zer0Kerbal/CargoBays) did it right. Do not change anything.

This issue should be probably closed as "invalid", as the problem is on Stock.

**Comment by Lisias** [comment]
I found one 2.5 part with problems. Fixed it on Pull Request https://github.com/zer0Kerbal/CargoBays/pull/53 .
